<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">margin:0px;
padding:0px;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="Editor">
        <item>
         <widget class="TextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="compileBtn">
          <property name="text">
           <string>compile</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <layout class="QHBoxLayout" name="rightside">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>2</number>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <layout class="QHBoxLayout" name="renderer_2">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="GLRendererWidget" name="renderer" native="true">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>128</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>2048</width>
                 <height>2048</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <layout class="QHBoxLayout" name="Hcompositor">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="handleWidth">
               <number>1</number>
              </property>
              <widget class="ModuleList" name="moduleListView">
               <property name="maximumSize">
                <size>
                 <width>128</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
              <widget class="OpComposerView" name="opcompositer">
               <property name="maximumSize">
                <size>
                 <width>2048</width>
                 <height>2048</height>
                </size>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_3">
           <layout class="QHBoxLayout" name="HLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="signatureParamsLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="TimelineView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>8</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadKernel"/>
    <addaction name="actionOpenImage"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuRendering"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadKernel">
   <property name="text">
    <string>load kernel...</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>open image...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Baracuda...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
   <property name="shortcut">
    <string extracomment="ctrl+w">Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpComposerView</class>
   <extends>QGraphicsView</extends>
   <header>opcomposer.h</header>
  </customwidget>
  <customwidget>
   <class>ModuleList</class>
   <extends>QListView</extends>
   <header location="global">modulelist.h</header>
  </customwidget>
  <customwidget>
   <class>GLRendererWidget</class>
   <extends>QWidget</extends>
   <header>glrendererwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimelineView</class>
   <extends>QGraphicsView</extends>
   <header>timelineview.h</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
